### PR TITLE
WIP Pm 2022 05 captcheck fix

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: adhocracy-plus/templates/account/signup.html [hunk 1]
```diff
@@ -53,6 +53,7 @@ <h1>{% trans "Register" %}</h1>
         {% endif %}
 
         {% include 'a4_candy_contrib/includes/form_field.html' with field=form.captcha %}
+        {{ settings.a4_candy_cms_settings.OrganisationSettings.contacts | richtext }}
```
Comment (phillimorland) [inline adhocracy-plus/templates/account/signup.html:56]: @fuzzylogic2000 This is not ideal but other uses of settings links in help-text were all done in the html, I tried getting it into the modal help text but not very hard as I'm not sure how to get the correct string out of the rich-text to suit the sentence anyway so I though this was the best bet but pretty sure i'm wrong, or we make a setting just for support?
Maybe I can use `bleach` and only allow the link? that will only work if email is always added as a `mailto` link which probably isn't a good idea?! err brain is foggy sorry!

File: apps/cms/contacts/models.py [hunk 2]
```diff
@@ -35,7 +35,9 @@ def formfields(self):
         # Add captcha to formfields property
         fields = super().formfields
         fields['captcha'] = CaptcheckCaptchaField(
-            label=_('I am not a robot'))
+            label=_('I am not a robot'),
+            help_text=_('If yu are having difficulty please contact us, '
```
Comment (goapunk) [inline apps/cms/contacts/models.py:39]: typo: yu -> you

File: apps/cms/contacts/models.py [hunk 3]
```diff
@@ -32,15 +34,22 @@ class FormField(AbstractFormField):
...
     @property
     def formfields(self):
+        contacts = self.get_contacts_for_default_site()
         # Add captcha to formfields property
         fields = super().formfields
         fields['captcha'] = CaptcheckCaptchaField(
             label=_('I am not a robot'),
             help_text=_('If you are having difficulty please contact us, '
-                        'details can be found adjacent.'))
+                        'details can be found adjacent.') + contacts)
```
Comment (Rineee) [inline apps/cms/contacts/models.py:43]: Yeah, so this is the richtext as str.. not sure if that helps?

Comment (phillimorland) [reply apps/cms/contacts/models.py:43]: ooooo, danke i'll give it a go

Comment (phillimorland) [reply apps/cms/contacts/models.py:43]: @Rineee dumb question to add this to the signup form I just change self to form or do I need to restructure the class?

File: apps/users/forms.py [hunk 4]
```diff
@@ -41,7 +41,11 @@ class DefaultSignupForm(SignupForm):
                     'additional information via email.'),
         required=False
     )
-    captcha = CaptcheckCaptchaField(label=_('I am not a robot'))
+    captcha = CaptcheckCaptchaField(
+        label=_('I am not a robot'),
+        help_text=_(
+            'If you are having difficulty please find our contact details '
+            'below.'))
```
Comment (phillimorland) [inline apps/users/forms.py:48]: This is where I wanted it, sorry it was form.py not modal

File: apps/cms/contacts/models.py (hunk 3 above)
Comment (phillimorland) [reply apps/cms/contacts/models.py:43]: NOTE: this solution retains all the html tags, would need to be cleaned first

**Pull request closed without merging**